bigtable_automation/gcf/custom_instance.go

```diff
@@ -84,35 +67,24 @@ func customInternal(ctx context.Context, e GCSEvent) error {
 		if err := setupBT(ctx, projectID, instance, tableID); err != nil {
 			return err
 		}
-		if numNodes < nodesH {
-			if err := scaleBT(ctx, projectID, instance, cluster, int32(nodesH)); err != nil {
-				return err
-			}
-		}
 		dataPath := joinURL(rootFolder, "cache")
 		controlPath := joinURL(rootFolder, "control")
 		err = launchDataflowJob(ctx, projectID, instance, tableID, dataPath, controlPath, dataflowTemplate)
 		if err != nil {
+			if errDeleteBT := deleteBTTable(ctx, projectID, instance, tableID); errDeleteBT != nil {
+				log.Printf("Failed to delete BT table on failed Dataflow launch: %v", errDeleteBT)
+			}
 			return err
 		}
 		// Save the fact that we've launched the dataflow job.
 		err = writeToGCS(ctx, launchedPath, "")
 		if err != nil {
+			if errDeleteBT := deleteBTTable(ctx, projectID, instance, tableID); errDeleteBT != nil {
+				log.Printf("Failed to delete BT table on failed GCS write: %v", errDeleteBT)
+			}
 			return err
 		}
 		log.Printf("[%s] State Launched", e.Name)
-	} else if strings.HasSuffix(e.Name, completedFile) {
-		log.Printf("[%s] State Completed", e.Name)
-		// Called when the state-machine moves to Completed state from Launched.
-		if numNodes == nodesH {
-			// Only scale down BT nodes when the current high node is set up this config.
-			// This requires different high nodes in different config.
-			if err := scaleBT(ctx, projectID, instance, cluster, int32(nodesL)); err != nil {
-				return err
-			}
-		}
-		// TODO: else, notify Mixer to load the BT table.
-		log.Printf("[%s] Completed work", e.Name)
```

Comment:
Can you restore the completed work message?

Reply:
done.